src/components/LoginRegister.vue

```diff
@@ -0,0 +1,207 @@
+                type="password"
+                name="password"
+                placeholder="Create password"
+                class="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
+                required=""
+              />
+            </div>
+            <div>
+              <label
+                for="confirm-password"
+                class="block mb-2 text-sm font-medium text-gray-900 dark:text-white"
+                >Confirm password</label
+              >
+              <input
+                id="confirm-password"
+                type="confirm-password"
+                name="confirm-password"
+                placeholder="Confirm new password"
+                class="bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-600 focus:border-primary-600 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
+                required=""
+              />
+            </div>
+            <div class="flex items-start">
+              <div class="flex items-center h-5">
+                <input
+                  id="terms"
+                  aria-describedby="terms"
+                  type="checkbox"
+                  class="w-4 h-4 border border-gray-300 rounded bg-gray-50 focus:ring-3 focus:ring-primary-300 dark:bg-gray-700 dark:border-gray-600 dark:focus:ring-primary-600 dark:ring-offset-gray-800"
+                  required=""
+                />
+              </div>
+              <div class="ml-3 text-sm">
+                <label
+                  for="terms"
+                  class="font-light text-gray-500 dark:text-gray-300"
+                  >I accept the
+                  <a
+                    class="font-medium text-primary-600 hover:underline dark:text-primary-500"
+                    href="#"
+                    >Terms and Conditions</a
+                  ></label
+                >
+                <div>
+                  <button
+                    type="submit"
+                    class="group relative flex w-full justify-center rounded-md border border-transparent bg-violet-500 py-2 px-4 text-sm font-medium text-white hover:bg-violet-700 focus:outline-none focus:ring-2 focus:ring-violet-500 focus:ring-offset-2 mt-5"
+                  >
+                    <span
+                      class="absolute inset-y-0 left-0 flex items-center pl-3"
+                    >
+                    </span>
+                    Register
+                  </button>
+                </div>
+              </div>
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import MainNavbar from "./MainNavbar.vue";
```

Comment:
If we show this page through the router view, it's not necessary to import the MainNavbar component here because it's already imported.

Reply:
You should not be able to see this! It is already fixed in my updated PR